website/bot.py

```diff
@@ -147,6 +147,9 @@ def load_vector_store():
 
 
 def conversation_chain(vector_store):
+    # not want to use magic numbers
+    search_kwargs = 5
```

Comment:
It's not clear from the name what's the meaning of this value. Could you make it more descriptive?

Reply:
@arkid15r  update done sir